### PR TITLE
disttask, ddl: rename flowhandle

### PR DESCRIPTION
**Pull request by ywqzzy**
### What problem does this PR solve?

Issue Number: ref #46258 

Problem Summary:

### What is changed and how it works?
Rename flowHandle interface to Dispatcher.
Raname processXXFlow.
Remove useless example.

Init Dispatcher interface when NewDispatcher then we can remove GetTaskDispatcher()/GetTaskFlowHandle.
### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: ddl/litBackfill_dispatcher.go
Comment (tangenta) [inline ddl/litBackfill_dispatcher.go:1]: It is better not to mention `lit`, I suggest `backfill_dispatcher.go`.

File: ddl/backfilling_dispatcher.go [hunk 1]
```diff
@@ -32,28 +32,28 @@ import (
...
 		return nil, errors.New("The getDDL result should be the type of *ddl")
 	}
-	return &litBackfillFlowHandle{
+	return &backfillingDispatcher{
 		d: ddl,
 	}, nil
 }
 
-func (*litBackfillFlowHandle) OnTicker(_ context.Context, _ *proto.Task) {
+func (*backfillingDispatcher) OnTicker(_ context.Context, _ *proto.Task) {
```
Comment (tangenta) [inline ddl/backfilling_dispatcher.go:52]: Rename this function to `OnTick`?

File: ddl/ddl.go [hunk 2]
```diff
@@ -690,11 +690,11 @@ func newDDL(ctx context.Context, options ...Option) *ddl {
 			return NewBackfillSchedulerHandle(ctx, taskMeta, d, step == proto.StepTwo)
 		})
 
-	backfillHandle, err := NewLitBackfillFlowHandle(d)
+	backFillDsp, err := NewBackfillingDispatcher(d)
 	if err != nil {
-		logutil.BgLogger().Warn("NewLitBackfillFlowHandle failed", zap.String("category", "ddl"), zap.Error(err))
+		logutil.BgLogger().Warn("NewBackfillingDispatcher ailed", zap.String("category", "ddl"), zap.Error(err))
```
Comment (tangenta) [inline ddl/ddl.go:695]:
```suggestion
		logutil.BgLogger().Warn("NewBackfillingDispatcher failed", zap.String("category", "ddl"), zap.Error(err))
```

File: ddl/backfilling_dispatcher.go (hunk 1 above)
Comment (ywqzzy) [reply ddl/backfilling_dispatcher.go:52]: > Rename this function to `OnTick`?

Ok

**Review by tangenta — APPROVED** [review]

**Review by xhebox — APPROVED** [review]

**Pull request merged**